winelist/winelist.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="ja">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>WineList</title>
+    <link href="style.css" rel="stylesheet" />
+  </head>
+  <body>
+    <div class="header"><h2>Wine List</h2></div>
+    <div id="selectors">
+      <input name="wines" value="Reds" id="reds" type="radio" checked />Reds
+      <input name="wines" value="Whites" id="whites" type="radio" />Whites
+      <input
+        name="wines"
+        value="Sparkling"
+        id="sparkling"
+        type="radio"
+      />Sparkling
+      <input name="wines" value="Dessert" id="dessert" type="radio" />Dessert
+      <input name="wines" value="Port" id="port" type="radio" />Port
```

Comment:
以下のようにすれば`winelist/winelist.js`のline1~5,line79~98を削除できそうに思います。

```suggestion
<input name="wines" value="Port" id="port" type="radio" onclick="fetchWines('port')" />Port
```

Reply:
参考コードありがとうございます！
ラジオボタンidを取得し引数に渡す方法も考えましたが、逆にコード量が増えたのでどう修正しようか悩んでいました。。
このように変更すれば、ラジオボタンを追加してもJS側の修正は不要になりますね。